### PR TITLE
Add a changelog message about the images moving

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -10,9 +10,14 @@ Informations
    * The package name become `{{package}}_geoportal`.
    * The package is in the folder `geoportal`.
 
-2. The `shortener.email_body` in the vars files use now the template syntax.
+2. All the image will be moved form `geoportal/{{package}}_geoportal/static-ngeo/images`
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: All the images will be moved from

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -10,9 +10,14 @@ Informations
...
+   to `geoportal/{{package}}_geoportal/static-ngeo/js/apps/image`, this is required for the apps.
+   If some images are used dirrectly by the apps (e.g. the theme icons) they should be moved back to they
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: dirrectly -> directly
to they -> to their

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 3]
```diff
@@ -10,9 +10,14 @@ Informations
...
+   original folder.
 
-3. Update access control list (ACL) in root context to allow user with role 'role_admin' to access views
+3. The `shortener.email_body` in the vars files use now the template syntax.
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: use now -> now uses

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 4]
```diff
@@ -10,9 +10,14 @@ Informations
...
+
+4. Update access control list (ACL) in root context to allow user with role 'role_admin' to access views
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: allow user -> allow users

**Comment by sbrunner** [comment]
Thanks @jwkaltz for the review :-)

**Pull request merged**